test/e2e/tests/account/snap-account-transfers.spec.ts

```diff
@@ -15,151 +16,309 @@ import HomePage from '../../page-objects/pages/homepage';
 import SnapSimpleKeyringPage from '../../page-objects/pages/snap-simple-keyring-page';
 import { installSnapSimpleKeyring } from '../../page-objects/flows/snap-simple-keyring.flow';
 import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
-import { sendTransactionWithSnapAccount } from '../../page-objects/flows/send-transaction.flow';
+import {
+  sendRedesignedTransactionWithSnapAccount,
+  sendTransactionWithSnapAccount,
+} from '../../page-objects/flows/send-transaction.flow';
 
 describe('Snap Account Transfers @no-mmi', function (this: Suite) {
-  it('can import a private key and transfer 1 ETH (sync flow)', async function () {
-    await withFixtures(
-      {
-        fixtures: new FixtureBuilder().build(),
-        ganacheOptions: multipleGanacheOptions,
-        title: this.test?.fullTitle(),
-      },
-      async ({
-        driver,
-        ganacheServer,
-      }: {
-        driver: Driver;
-        ganacheServer?: Ganache;
-      }) => {
-        await loginWithBalanceValidation(driver, ganacheServer);
-        await installSnapSimpleKeyring(driver);
-        const snapSimpleKeyringPage = new SnapSimpleKeyringPage(driver);
-
-        // import snap account with private key on snap simple keyring page.
-        await snapSimpleKeyringPage.importAccountWithPrivateKey(
-          PRIVATE_KEY_TWO,
-        );
-        await driver.switchToWindowWithTitle(
-          WINDOW_TITLES.ExtensionInFullScreenView,
-        );
-        const headerNavbar = new HeaderNavbar(driver);
-        await headerNavbar.check_accountLabel('SSK Account');
-
-        // send 1 ETH from snap account to account 1
-        await sendTransactionWithSnapAccount({
+  describe('Old confirmation screens', function () {
```

Comment:
Since both test the old and new `describe`'s are very similar:

```diff
1c1
<   describe('Redesigned confirmation screens', function () {
---
>   describe('Old confirmation screens', function () {
17a18,19
>           await tempToggleSettingRedesignedTransactionConfirmations(driver);
>
32c34
<           await sendRedesignedTransactionWithSnapAccount({
---
>           await sendTransactionWithSnapAccount({
35a38,39
>             gasFee: '0.000042',
>             totalFee: '1.000042',
64a69,70
>           await tempToggleSettingRedesignedTransactionConfirmations(driver);
>
79c85
<           await sendRedesignedTransactionWithSnapAccount({
---
>           await sendTransactionWithSnapAccount({
82a89,90
>             gasFee: '0.000042',
>             totalFee: '1.000042',
113a122,123
>           await tempToggleSettingRedesignedTransactionConfirmations(driver);
>
128c138
<           await sendRedesignedTransactionWithSnapAccount({
---
>           await sendTransactionWithSnapAccount({
131a142,143
>             gasFee: '0.000042',
>             totalFee: '1.000042',
```

I would prefer to have something like a `describe.each` to avoid repetition as much as we can.

So something like this maybe:
```ts
describe.each([
  {
    desc: 'Old',
    setup: async (driver: Driver) => {
      await tempToggleSettingRedesignedTransactionConfirmations(driver);
    },
    sendTransaction: async (driver: Driver) => {
      await sendRedesignedTransactionWithSnapAccount({
        driver,
        recipientAddress: DEFAULT_FIXTURE_ACCOUNT,
        amount: '1',
      });
    },
  },
  {
    desc: 'New',
    setup: async () => {
      // Nothing to setup for new confirmations.
    },
    sendTransaction: async (driver: Driver) => {
      await sendTransactionWithSnapAccount({
        driver,
        recipientAddress: DEFAULT_FIXTURE_ACCOUNT,
        amount: '1',
        gasFee: '0.000042',
        totalFee: '1.000042',
      });
    },
  },
])('$desc confirmation screens', ({ setup, sendTransaction }) => {
  it('can import a private key and transfer 1 ETH (sync flow)', async function () {
    await withFixtures(
        {
          fixtures: new FixtureBuilder().build(),
          ganacheOptions: multipleGanacheOptions,
          title: this.test?.fullTitle(),
        },
        async ({
          driver,
          ganacheServer,
        }: {
          driver: Driver;
          ganacheServer?: Ganache;
        }) => {
          await loginWithBalanceValidation(driver, ganacheServer);

          await setup(driver);

          await installSnapSimpleKeyring(driver);
          ...
          // Also use `sendTransaction` instead of `sendTransactionWithSnapAccount` and `sendRedesignedTransactionWithSnapAccount`.
          ...
    
})
```

Reply:
After discussing internally with @pedronfigueiredo, the "Old confirmation screens" are going to be removed "soon", so I'm ok with keeping the duplicate as-is, as long as we have a `TODO` here to mark this code-scope and make sure we're gonna remove it somedays.

Thanks!

```suggestion
  // TODO: Remove the old confirmations screen tests once we have fully migrated
  // to the new ones.
  describe('Old confirmation screens', function () {
```